.goreleaser.yaml

```diff
@@ -1,31 +1,39 @@
-# .goreleaser.yaml
 builds:
-- skip: true # skip build for now since we have another mechanism to build all kubestellar binaries
-  id: kubestellar-syncer
-  main: ./cmd/syncer
-  binary: kubestellar-syncer
+- id: "controller-manager"
+  main: ./cmd/controller-manager
+  binary: bin/controller-manager
+  ldflags:
+  - "{{ .Env.LDFLAGS }}"
   goos:
   - linux
-  - darwin
   goarch:
   - amd64
   - arm64
-  - s390x
+  ignore:
+  - goos: darwin
+    goarch: ppc64le
+  env:
+  - CGO_ENABLED=0
+kos:           
+  - repository: ghcr.io/kubestellar/kubestellar/controller-manager
+    main: ./cmd/controller-manager
+    build: controller-manager
+    tags:
+    - '{{.Version}}'
+    - latest
+    bare: true
+    preserve_import_paths: false
+    ldflags:
+    - "{{ .Env.LDFLAGS }}"
+    platforms:
+    - linux/amd64
+    - linux/arm64
+    env:
+    - CGO_ENABLED=0
+release:
+  disable: true # disable release while testing publishing 
```

Comment:
Is it true that the real reason that we disable publishing GitHub "release artifacts" is that we have none to publish now?

Reply:
Note also that when we use a tag with a pattern `v${major}.${minor}.${patch}-{something}` goreleaser creates a pre-release, and if the tag has the pattern `v${major}.${minor}.${patch}` then go release creates a release.  That is the result of using  `prerelease: auto` - see https://goreleaser.com/customization/release/#github